ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -138,7 +141,12 @@ func ConvertToTaskDefinition(params ConvertTaskDefParams) (*ecs.TaskDefinition,
 	if params.RequiredCompatibilites != "" {
 		taskDefinition.RequiresCompatibilities = []*string{aws.String(params.RequiredCompatibilites)}
 	}
-
+	if taskDefParams.pidMode != "" {
```

Comment:
I would move this logic into the `entity_helper` (see how `networkMode`, `cpu` etc is done), to ensure better default behavior for no values being set.

Reply:
Personally I like having it here better, since we have better tests for this package and it allows you to test that its empty. For me at least, its easier to understand if its done here.

But I'm open to changing ientity_helper as well if you feel strongly about it.